.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: ruby
```

Comment:
@Naros I think the language should be switched to https://docs.travis-ci.com/user/languages/minimal-and-generic/ as the build is delegated to the Docker